src/utils/get-ts-config.js

```diff
@@ -51,6 +52,26 @@ const getTsConfig = async (memo, fs, sourcePath, projectRoot) => {
     }
 };
 
+/**
+ * @param {string} jsoncString
+ * @param {string?} fileName
+ * @returns {any}
+ */
+function parseJsoncOrThrow(jsoncString, fileName){
```

Comment:
I think a separate file like `utils/parse-jsonc.js` would work better for this function